test/core/libp2p/peer/protocol_repository/inmem_protocol_repository_test.cpp

```diff
@@ -0,0 +1,151 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <gtest/gtest.h>
+
+#include "common/buffer.hpp"
+#include "libp2p/peer/errors.hpp"
+#include "libp2p/peer/protocol_repository.hpp"
+#include "libp2p/peer/protocol_repository/inmem_protocol_repository.hpp"
+#include "testutil/outcome.hpp"
+
+using namespace libp2p::peer;
+using namespace libp2p::multi;
+using kagome::common::Buffer;
+
+struct InmemProtocolRepository_Test : public ::testing::Test {
+  std::unique_ptr<ProtocolRepository> db;
```

Comment:
Add _ postfix as it is a field

Reply:
same, could be applied for others fields as well

Reply:
it is public